core/winibuild.py

```diff
@@ -0,0 +1,81 @@
+"""Builds wandb-core."""
+
+import os
+import pathlib
+import subprocess
+from typing import Mapping
+
+from tools.wini import workspace
+
+
+def build_nexus(
```

Comment:
```suggestion
def build_wandb_core(
```

Reply:
Done -- had to make change in wini.py as well.